.github/workflows/build.yml

```diff
@@ -195,8 +195,23 @@ jobs:
         uses: docker/login-action@v2
         with:
           registry: docker.io
-          username: ${{ steps.string_user.outputs.lowercase }}
+          username: ${{ vars.DOCKER_USERNAME || steps.string_user.outputs.lowercase }}
           password: ${{ secrets.DOCKER_TOKEN }}
+      - name: Set Docker name with owner (package name) depending on if DOCKER_USERNAME is set
+        if: ${{ steps.vars.outputs.HAS_SECRET_TOKEN }}
+        id: docker_repo
+        shell: bash
+        run: |
+          if [ -z $DOCKER_USERNAME ]; then
+            echo "DOCKER_NWO=${GITHUB_NWO}" >> $GITHUB_OUTPUT
+          else
+            IFS='/' read -ra NWO_SPLIT <<< "$GITHUB_NWO"
+            REPO_NAME=${NWO_SPLIT[1]}
+            echo "DOCKER_NWO=${DOCKER_USERNAME}/${REPO_NAME}" >> $GITHUB_OUTPUT
+          fi
+        env:
+          DOCKER_USERNAME: ${{ vars.DOCKER_USERNAME }}
+          GITHUB_NWO: ${{ steps.string_repo.outputs.lowercase }}
```

Comment:
Why split it here. Why not use `string_user`

Reply:
Isn't it `github.repository_owner` I'm using it above (after conversion to lowercase coz docker hub wanted lower case when I registered) for my username.

Reply:
Oh wait your after the repo name?

Reply:
I think that's what I get for checking code while travelling

Reply:
right - basically just the repo name to combine with the docker username